AsyncDisplayKit/ASButtonNode.mm

```diff
@@ -491,10 +491,17 @@ - (ASLayoutSpec *)layoutSpecThatFits:(ASSizeRange)constrainedSize
     spec = [ASInsetLayoutSpec insetLayoutSpecWithInsets:contentEdgeInsets child:spec];
   }
   
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wdeprecated-declarations"
   if (CGSizeEqualToSize(self.preferredFrameSize, CGSizeZero) == NO) {
-    stack.sizeRange = ASRelativeSizeRangeMakeWithExactCGSize(self.preferredFrameSize);
+#if DEBUG
```

Comment:
Why do we still need to use the preferredFrameSize implementations here? We should remove all usage of it in the internal components to support the fact it's deprecated and we ourselves don't depend on it to exist.